app/forms/questionnaire_form.py

```diff
@@ -128,7 +129,11 @@ def validate_mutually_exclusive_question(self, question):
         try:
             validator(answers, is_mandatory)
         except validators.ValidationError as e:
-            self.question_errors[question["id"]] = str(e)
+            error_message = str(e)
+
+            self.question_errors[question["id"]] = FieldHandler.format_question_title(
+                error_message, question.get("title")
+            )
```

Comment:
If we made `question_title` a property of questionnaire_form, we could assign it the safe value, and remove the need to do it here and in the field handler.

```suggestion
            self.question_errors[question["id"]] = error_message % dict(question_title=self.question_title)
            )
```